deployment/migrations/versions/0024__update_price_instance.py

```diff
@@ -0,0 +1,158 @@
+                              sum(vm_machine_volumes.size_mib) * 1024 * 1024 AS other_volumes_size
+                       FROM vm_machine_volumes
+                       GROUP BY vm_machine_volumes.vm_hash) other_volumes_size
+                      ON vm_versions.current_version::text = other_volumes_size.vm_hash::text,
+             LATERAL ( SELECT file_volumes_size.file_volumes_size +
+                              other_volumes_size.other_volumes_size::numeric AS required_disk_space) used_disk,
+             LATERAL ( SELECT ceil(GREATEST(ceil((vms.resources_vcpus / 1)::double precision),
+                                            (vms.resources_memory / 2048)::double precision)) AS compute_units_required) cu,
+             LATERAL ( SELECT CASE
+                                  WHEN COALESCE(vms.persistent, true)
+                                      THEN '21474836480'::bigint::double precision * cu.compute_units_required
+                                  ELSE '2147483648'::bigint::double precision * cu.compute_units_required
+                                  END AS included_disk_space) free_disk,
+             LATERAL ( SELECT GREATEST((file_volumes_size.file_volumes_size + rootfs_size.rootfs_size::numeric +
+                                        other_volumes_size.other_volumes_size::numeric)::double precision -
+                                       free_disk.included_disk_space,
+                                       0::double precision) AS additional_disk_space) additional_disk,
+             LATERAL ( SELECT CASE
+                                  WHEN vms.payment_type = 'superfluid' THEN 0
+                                  WHEN COALESCE(persistent, true)
+                                    AND environment_trusted_execution_policy IS NOT NULL 
+                                    AND environment_trusted_execution_firmware IS NOT NULL THEN 2000
+                                  WHEN COALESCE(persistent, true) THEN 1000
+                                  ELSE 200
+                                  END AS base_compute_unit_price) bcp,
+             LATERAL ( SELECT CASE
+                                  WHEN COALESCE(vms.persistent, true) THEN 1 + vms.environment_internet::integer
+                                  ELSE 1
+                                  END AS compute_unit_price_multiplier) m,
+             LATERAL ( SELECT cu.compute_units_required * m.compute_unit_price_multiplier::double precision *
+                              bcp.base_compute_unit_price::double precision AS compute_unit_price) cpm,
+             LATERAL ( SELECT CASE
+                        WHEN vms.payment_type = 'superfluid' THEN 0
+                        ELSE additional_disk.additional_disk_space / 20::double precision / (1024 * 1024)::double precision
+                        END AS disk_price
+             ) adp,
+             LATERAL ( SELECT cpm.compute_unit_price + adp.disk_price AS total_price) tp
+        """
+
+    )
```

Comment:
This is probably for historical reasons but I'm surprised we used giga raw sql to handle sql view while we are using sqlalchemy '-'

I guess we aren't left with that many other choices but I'm concerned about the maintability of such approach :/